warehouse/templates/search/results.html

```diff
@@ -84,6 +84,32 @@ <h3 class="package-snippet__title">
 {% endif %}
 {%- endmacro %}
 
+
+
+{% macro generate_markup(top_level, classifiers) -%}
+    {% set _id = top_level %}
+    {%- for classifier in classifiers %}
+     {% set children = classifiers[classifier] %}
+     {% if kwargs['parent'] != None %}
+      {% set _id = _id +' :: '+ kwargs['parent'] %}
```

Comment:
could you rename `_id` to something like `level_id`? it would be clearer what the purpose of it is in the line below